tests/test_pipeline_media.py

```diff
@@ -54,54 +68,6 @@ def tearDown(self):
             if not name.startswith("_"):
                 disconnect_all(signal)
 
-    def test_default_media_to_download(self):
-        request = Request("http://url")
-        assert self.pipe.media_to_download(request, self.info) is None
-
-    def test_default_get_media_requests(self):
-        item = {"name": "name"}
-        assert self.pipe.get_media_requests(item, self.info) is None
-
-    def test_default_media_downloaded(self):
-        request = Request("http://url")
-        response = Response("http://url", body=b"")
-        assert self.pipe.media_downloaded(response, request, self.info) is response
-
-    def test_default_media_failed(self):
-        request = Request("http://url")
-        fail = Failure(Exception())
-        assert self.pipe.media_failed(fail, request, self.info) is fail
-
-    def test_default_item_completed(self):
```

Comment:
This is also useful and should be kept.

Reply:
Ah yes you are right ! 
Still trying to understand the project :)